support-frontend/assets/pages/supporter-plus-landing/setup/legacyReducer.ts

```diff
@@ -0,0 +1,103 @@
+// ----- Imports ----- //
```

Comment:
This is [contributionsLandingReducer.ts](https://github.com/guardian/support-frontend/blob/9d8786c148917d6a399dd5c4a6ccd7ea1b960531/support-frontend/assets/pages/contributions-landing/contributionsLandingReducer.ts), with unused state fields removed.